client/wp-admin-scripts/print-shipping-label-banner/dismiss-modal/index.js

```diff
@@ -16,13 +16,18 @@ export default class DismissModal extends Component {
 		onCloseAll();
 		// TODO: Maybe set counter for when to show again, or just show on next page load.
 
-		// TODO: tracking
+		this.props.trackBannerEvent(
+			'shipping_banner_dismiss_modal_remind_me_later_click'
+		);
 	};
 
 	closeForeverClicked = () => {
 		const { onCloseAll } = this.props;
 		// TODO: Persist something so the banner never appears again.
 		onCloseAll();
+		this.props.trackBannerEvent(
```

Comment:
I think it would be cleaner to bring in trackBannerEvent on the first line in this function instead of referencing it from this.props directly:
```
 		const { onCloseAll, trackBannerEvent } = this.props; 
```

Reply:
I agree!